src/datasets/data_utils.py

```diff
@@ -114,6 +127,7 @@ def index_as_one_hot(indices, axis=0, dtype=np.int32):
     return onehot_encoding
 
 def to_embedded_word(txt):
+    from word_vector_embedder import WordVectorEmbedder
     embedder = WordVectorEmbedder('glove')
     return embedder.embed_words_into_vectors(text)
```

Comment:
Typo with function taking in "txt" as parameter and returning function on "text"